app/src/main/java/co/nimblehq/alpine/sample/ui/camera/CameraCaptureActivity.kt

```diff
@@ -172,29 +177,25 @@ class CameraCaptureActivity : ComponentActivity() {
                         }
 
                         override fun onImageSaved(output: ImageCapture.OutputFileResults) {
-                            processImage(photoFile)
+                            viewModel.processImageFile(filePath = photoFile.absolutePath)
                         }
                     })
             }
         }
     }
 
-    private fun processImage(photoFile: File) {
-        mrzProcessor.processImageFile(photoFile.absolutePath, object : MrzProcessorResultListener {
-            override fun onSuccess(mrzInfo: MrzInfo) {
-                loadingDialog.dismiss()
-                NfcScanningActivity.start(this@CameraCaptureActivity, mrzInfo)
-            }
+    private fun navigateToNfcScanning(mrzInfo: MrzInfo) {
+        loadingDialog.dismiss()
+        imageAnalysis?.clearAnalyzer()
```

Comment:
TODO: Need to update `imageAnalysis?.clearAnalyzer()` to make sure it was clear before navigate to `NfcScanScreen`